cosmwasm/contracts/crosschain-swaps/src/contract.rs

```diff
@@ -51,25 +51,30 @@ pub fn execute(
     info: MessageInfo,
     msg: ExecuteMsg,
 ) -> Result<Response, ContractError> {
+    // IBC transfers support only one token at a time
     match msg {
         ExecuteMsg::OsmosisSwap {
-            input_coin,
+            swap_amount,
```

Comment:
nit: consider just `amount`

Reply:
I was worried this would be confused with the minAmountOut parameter of the slippage (and maybe later with other params), but maybe I'm worrying about nothing?